src/Makefile.am

```diff
@@ -56,6 +56,9 @@ WordBreakProperty.txt:
 GraphemeBreakProperty.txt:
 	wget http://www.unicode.org/Public/UNIDATA/auxiliary/GraphemeBreakProperty.txt
 
+emoji-data.txt:
+	wget https://www.unicode.org/Public/emoji/11.0/emoji-data.txt
```

Comment:
Should we use a version-independent path here?

Reply:
I would like to, but I could not find a version independent path to that file

Reply:
In that case, we would have to live with it… 😂